src/Files/allEvents.js

```diff
@@ -1,4 +1,73 @@
-export default [
+import jQuery from 'jquery';
+
+window.$ = window.jQuery = jQuery;
+
+let eventlist = []
+
+// Stuff to hook it up to the Google Cloud Api, API_KEY are unique
+// Generating an API Key: https://developers.google.com/workspace/guides/create-project
+const API_KEY = 'AIzaSyDmgrkOrEu0ZFMxT8ra1H42evtCDoKXhA8';
+
+// This needs to be set to a PUBLIC Google calendar
+// Also this type of access is read-only so you can't push events from the cs club website if you wanted to try that
+// How to get calendar ID: https://yabdab.zendesk.com/hc/en-us/articles/205945926-Find-Google-Calendar-ID
+const CAL_ID = '25h073198b7tpg7qcj6uhu1t8k@group.calendar.google.com';
+
+let months = ["JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC"]
+let newEvents = [];
+
+    jQuery.ajax({
+        url:"https://www.googleapis.com/calendar/v3/calendars/" + CAL_ID + "/events?singleEvents=true&orderBy=startTime&sortOrder=ascending&timeMin=" + (new Date()).toISOString() + "&key=" + API_KEY,
+        type: "GET",
+        dataType: "json",
+        async:false,
+        // async lets the data stay saved after the inital call, otherwise the data will be removed after the success call
+        success: function(data) {
+          console.log(data);
+          eventlist = data.items;
+          console.log(eventlist);
+        }
+          });
+
+
+          for(var i = 0; i < eventlist.length; i++){
+            let hours = parseInt(eventlist[i].start.dateTime.substring(11, 13));
+            let suffix = "";
+
+            if (hours >= 12) {
+                suffix = "PM";
+                hours %= 12;
+            }
+            else {
+                suffix = "AM";
+                if (hours == 0) {
+                    hours = 12;
+                }
+            }
+            let time = hours.toString() + eventlist[i].start.dateTime.substring(13, 16) + " " + suffix;
```

Comment:
This is good for now since the datetime string returned from the HTTP request is formatted properly, but generally speaking hard coding is not the best practice (referring to substring(11,13) and substring(13,16) in particular). To make sure that the string parsing works on any string you may consider use a Date object's built-in function to parse the string or split it by pattern rather than index. Like the string "20210728T12:00", you can split it by split("T") into two parts.